### ISSUE TITLE
EOFError: marshal data too short

### ISSUE THREAD

**Issue by oroppas** [oracle/graalpython#229]
```
graalpython -m pip list
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/ryuta/packages/graal/graal/sdk/mxbuild/linux-amd64/GRAALVM_EA311E5C9C_JAVA11/graalvm-ea311e5c9c-java11-21.3.0-dev/languages/python/lib-python/3/runpy.py", line 185, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/home/ryuta/packages/graal/graal/sdk/mxbuild/linux-amd64/GRAALVM_EA311E5C9C_JAVA11/graalvm-ea311e5c9c-java11-21.3.0-dev/languages/python/lib-python/3/runpy.py", line 144, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/home/ryuta/packages/graal/graal/sdk/mxbuild/linux-amd64/GRAALVM_EA311E5C9C_JAVA11/graalvm-ea311e5c9c-java11-21.3.0-dev/languages/python/lib-python/3/runpy.py", line 111, in _get_module_details
    __import__(pkg_name)
EOFError: marshal data too short
```

**Comment by msimacek** [comment]
Hi @oroppas, we forgot to bump the serialization version for our `pyc` files, sorry about that. There's a fix already in progress. For now, you can work around the problem by deleting the `pyc` files (`find -name '*.pyc' -delete`)

**Comment by oroppas** [comment]
Thanks for quick response, @msimacek. Good to know the fix is coming soon!

**Comment by oroppas** [comment]
@msimacek This issue seems to be fixed by f2f017bdd3cbabed8faed0c434e0c930f6ad604b on my side.

**Comment by msimacek** [comment]
Should be fixed by c5de6de4ef7d1ae6d47654e811a64ed6ffc8d30b
